### ISSUE TITLE
Keyboard shortcut conflict in JAWS screen reader on Windows

### ISSUE THREAD

**Issue by TaF-WCAG** [edrlab/readium-desktop#3510]
Keyboard shortcut: Shift Control k does not produce identical results with different reading methods. 

macOS 26.4.1 - standard laptop keyboard - visual in Thorium 3.4.0. The command provides location data in a small window at the bottom of the screen revealing the page number, chapter, and progression. 

Windows 11 - standard laptop keyboard - JAWS-2026 - in laptop keyboard mode - Thorium 3.4.0. The command places a place marker with list view and some options just like Chrome, it does not reveal a page number. 

We need a consistent shortcut or method to reveal the page number for both Screen Reader and Visual users.

**Comment by danielweck** [comment]
Hello, thank you for reporting this (mis)behaviour.

The `CTRL` + `SHIFT` + `K` keyboard shortcut is specifically designed to invoke a command that displays a "toast" popup notification which triggers a screen reader announcement (ARIA live=assertive) of the "current reading location", which includes the nearest preceding page break in the document (if any).

I do not understand your description: "The command places a place marker with list view and some options just like Chrome". Could you please explain? Thank you.

Code references:

https://github.com/edrlab/thorium-reader/blob/96c5c00c059c1a958b1f54f18bde43684dfcbb43/src/common/keyboard.ts#L57-L64

https://github.com/edrlab/thorium-reader/blob/96c5c00c059c1a958b1f54f18bde43684dfcbb43/src/renderer/reader/components/Reader.tsx#L1734-L1907

**Comment by danielweck** [comment]
> The command places a place marker with list view and some options just like Chrome

... follow-up: now I think I understand what you meant. One possibility is that the Thorium Desktop reader app doesn't even receive the keyboard shortcut callback, because the Windows operating system itself intercepts this specific sequence of key strokes to trigger a completely different action. If not the Windows OS, then it is quite possible that the JAWS screen reader software interferes with Thorium's keyboard shortcut preset.

**Comment by TaF-WCAG** [comment]
I queried the screen reader user and sounds like you have hit on the issue. From the user: 

I would say, when in a browser, the keyboard shortcut Ctl+shift+k brings up a dialogue box consisting of place markers placed on the page by the user, along with further options on adding or interacting with those place markers. Place markers allow users to mark areas of importance on a webpage by using the keyboard shortcut Ctl+Windows key+k. So, the shortcuts mentioned behave in the same way in Thorium as it does in a browser. This is when virtual curser is on.
When the virtual curser is off, Ctl+shift+k does announce some positioning, e.g. “Current page 20 – progression=6/8” followed by the top heading I think. Then, 17% --” followed by a list of the headings in that chapter, if I am understanding the layout correctly.    
 
So, constantly switching between virtual curser and PC curser seems to be key, even though it is laborious and frustrating – it does seem to be the way everything is going.

**Comment by danielweck** [comment]
This is very useful feedback, thank you. I am going to rename this issue (change the title) to describe the problem more accurately: there may be keyboard shortcuts that conflict with the operating system, the screen reader, or any other assistive technology. CTRL SHIFT K is one of these problematic shortcuts with JAWS, but perhaps not NVDA? This class of problem exists on MacOS for sighted users too (i.e. outside of Voice Over, just because of reserved / high-precedence system shortcuts, such as the window manager's own resize/move actions, just to name one concrete example). The keyboard presets in Thorium Desktop are the same across Linux, Windows and MacOS so there might be cases we've overlooked due to lack of testing with all possible window managers and assistive technologies. Keyboard shortcuts are configurable but this is of course not an ideal solution, it is a workaround.
